### PR TITLE
fix: dayjs locale format seconds is not working and is polluting the global dayjs locale

### PR DESCRIPTION
**Pull request by nikosrossolatos**
# Breaking Changes
N/A

# Changes

- fix: `getRelativeDateFromNow` will no longer return `%s sec` when seconds are provided.
  - dayjs@1.11.10 is expecting `"%d sec"` instead of `"%s sec"` ([here's why](https://github.com/iamkun/dayjs/blob/dev/src/plugin/relativeTime/index.js#L58)) to make the proper conversion to "10 sec"
- fix: do not pollute the global `en` locale 
  - We found out this bug while using this library on our app. It shouldn't be affecting _every_ app or library that uses this

### CODE REVIEW

File: packages/common/src/utils/DateUtil.ts [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 
-dayjs.updateLocale('en', {
+const localeObject = {
+  ...englishLocale,
+  name: 'en-custom',
   relativeTime: {
     future: 'in %s',
     past: '%s ago',
-    s: '%s sec',
+    s: '%d sec',
```
Comment (nikosrossolatos) [inline packages/common/src/utils/DateUtil.ts:15]: This is the main fix

File: packages/common/src/utils/DateUtil.ts [hunk 2]
```diff
@@ -21,14 +24,16 @@ dayjs.updateLocale('en', {
...
+dayjs.locale('en-custom', localeObject)
 
 export const DateUtil = {
   getYear(date: string = new Date().toISOString()) {
     return dayjs(date).year()
   },
 
   getRelativeDateFromNow(date: string | number) {
-    return dayjs(date).fromNow(true)
+    return dayjs(date).locale('en-custom').fromNow(true)
```
Comment (nikosrossolatos) [inline packages/common/src/utils/DateUtil.ts:37]: This is how I would propose to fix the global pollution of the dayjs en locale. Happy to try other solutions as well but I think this clean enough

File: packages/common/src/utils/DateUtil.ts [hunk 3]
```diff
@@ -21,14 +24,16 @@ dayjs.updateLocale('en', {
...
+dayjs.locale('en-web3-modal', localeObject)
 
 export const DateUtil = {
   getYear(date: string = new Date().toISOString()) {
     return dayjs(date).year()
   },
 
   getRelativeDateFromNow(date: string | number) {
-    return dayjs(date).fromNow(true)
+    return dayjs(date).locale('en-web3-modal').fromNow(true)
```
Comment (nikosrossolatos) [inline packages/common/src/utils/DateUtil.ts:37]: This is how I would propose to fix the global pollution issue. Happy to try other solutions but I think this is clean enough

**Review by enesozturk — APPROVED** [review]
LGTM

**Comment by nikosrossolatos** [comment]
> LGTM

@enesozturk How can I run the vercel or the rest of the checks since I am missing authorisation so that we can merge this PR? :D

**Comment by enesozturk** [comment]
@nikosrossolatos seems interesting, I'll investigate this and let you know. The issues are not expected so it shouldn't be a problem for people outside of the team, it's something we should fix unless you touch the GH workflow code.

**Comment by nikosrossolatos** [comment]
> Vercel ones need to be authorized. I authorized labs, if this is good we can just merge it

Yeap that's what it saying for me too 😅 missing authorisation. I'll leave this to you then to merge if it's okay

**Review by tomiir — APPROVED** [review]

**Review by tomiir — APPROVED** [review]

**Pull request merged**